server/public/model/scheduled_post.go

```diff
@@ -19,10 +19,6 @@ func (s *ScheduledPost) IsValid(maxMessageSize int) *AppError {
 		return NewAppError("ScheduledPost.IsValid", "model.scheduled_post.is_valid.draft.app_error", nil, "", http.StatusBadRequest).Wrap(draftAppErr)
 	}
 
-	if len(s.Message) == 0 {
-		return NewAppError("ScheduledPost.IsValid", "model.scheduled_post.is_valid.empty_message.app_error", nil, "id="+s.Id, http.StatusBadRequest)
-	}
-
```

Comment:
Do we want to completely remove this, or do we want to check the two conditions? (i.e. `len(s.Message) == 0 && len(s.FileIds) == 0)`)

Reply:
I'm comparing the behaviour of drafts and post and update accordingly.